scripts/verify_oracle.sh

```
@@ -0,0 +1,224 @@
+    if eval "$command"; then
+        echo "✅ PASSED: $test_name"
+        return 0
+    else
+        echo "❌ FAILED: $test_name"
+        return 1
+    fi
+}
+
+# Test 1: Check decimals() function
+test_decimals() {
+    local result=$(stellar contract invoke \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)